docker/dropin_env_base/requirements.txt

```diff
@@ -1,2 +1,2 @@
 datarobot-drum==1.10.14
-datarobot-mlops==9.2.8
+datarobot-mlops==10.0.3a2
```

Comment:
Is it ok to use it in pair with Java mlops-agent==9.2.8 ?

This is the agent installed on this envs. It is installed in the base images.

Reply:
It _should_ be ok in this instance because we haven't really done much (if any?) agent API changes this release. But it would be safer if we kept things at the same major version.

Reply:
But I see the latest version of mlops-agent on Maven is 9.2.8

Reply:
Yeah so while in theory this should work for these version skews, I'm pretty sure in actuality we don't support doing such a thing: https://github.com/datarobot/tracking-agent/blob/187a4bc83c56cd638a7921e380a319ba14a534d2/mlops-py/datarobot_mlops/agent.py#L438-L443

I think I'm going to have to go back to a prior change I had where instead of bumping the version and locking into the new import path, I can make a more backward/forward compatible change (but it's a bit uglier).